src/main/java/seedu/address/logic/commands/AddModuleCommand.java

```diff
@@ -0,0 +1,112 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.commands.DeleteModuleCommand.MESSAGE_FAILURE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.person.Status;
+import seedu.address.model.tag.Tag;
+
+public class AddModuleCommand extends Command {
+
+    public static final String COMMAND_WORD = "addmodule";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Add specified modules to the person identified "
+            + "by the index number used in the displayed person list. "
+            + "Input values will be added on to existing modules.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "[" + PREFIX_TAG + "TAG]...\n"
```

Comment:
May be a good time to consider changing all `TAG` to `MODULE`

Reply:
Agreed, will perform the project-wide refactor in a future commit